cwm-data/src/main/java/org/cast/cwm/service/UserService.java

```diff
@@ -153,7 +153,14 @@ public IModel<User> getBySubjectId (String subjectId) {
 		c.setSubjectId(subjectId);
 		return new HibernateObjectModel<User>(User.class, c);
 	}
-	
+
+	@Override
+	public IModel<User> getByLtiId (String ltiId) {
+		UserCriteriaBuilder c = new UserCriteriaBuilder();
```

Comment:
This method probably needs to be `getBySiteAndLtiId( Site site, String ltiId)` since LTI IDs are unique only within their Site.

Reply:
The other comments about prefixing.